docs/user-guide/data.md

```diff
@@ -183,15 +183,61 @@ If you don't have the permissions you were supposed to get, please contact the [
 
-<!--In time-based resources like moving images or audio documents, you can mark sequences on the timeline. A transcription tool helps to annotate and to transcribe the sequence.-->
+![Single resource view. The resource type in this example is "Page".](../assets/images/resource-view-with-image.png)*Single resource view. The resource type in this example is "Page" and an image viewer displays the photograph of the manuscript page.*
 
-![Single resource view. The resource type in this example is "Photo".](../assets/images/resource-view-with-image.png)*Single resource view. The resource type in this example is "Page".*
+Everything about the resource is displayed below the media viewer (when there is one). All the information about the resource itself (e.g. type, label, permissions, creator, date of creation, etc.), the data (properties and values), and different functionalities such as editing, deleting, sharing the resource. Get more details about the functionalities in the following sections.
 
-Additionally, you can work on the resource directly, e.g, mark regions of interest on still images and on documents.
+![Resource toolbar description.](../assets/images/resource-toolbar.png)
+
+##### Audio file
+
+Supported format: **mp3**
+
+![Audio viewer](../assets/images/audio-file.png)
+
+##### Image file
+
+Supported format: **jpeg**, **jp2**, **tiff**, **tiff-x**, **png**
+
+In a resource of type "still image", you're able to draw regions on the image and annotate this region. Usually, a still image resource is used for book pages, photographs, postcards, letters, etc.
+
+When you open a resource that contains an image, this one is displayed in a viewer. Several functionalities are accessible from the image viewer, e.g. zoom in/out, copy the IIIF link, replace the image or draw a region on the image.
+
+<!-- ![Image viewer](../assets/images/region_annotation_panel.png)*Image viewer toolbar.* -->
+![Image viewer](../assets/images/image-file.png)
+
+To access the regions, go to the **annotation** panel. You can **click on an existing region** in the image viewer, the focus of the page will be redirected to the annotation information.
+
+![Access the annotation panel](../assets/images/access_annotation_panel.png)*Access the annotation panel from the resource toolbar.*
+
+##### Document file
+
+Supported format: **PDF**
+
+![PDF viewer](../assets/images/pdf-file.png)
+
+##### Text file
+
+Supported format: **csv**, **txt**, **xml**
+
+![Text viewer](../assets/images/text-doc-file.png)
+
+##### Video file
+
+Supported format: **mp4**
+
+![Video viewer](../assets/images/video-file.png)
+
+##### Archive file
+
+Supported format: **zip**, **x-tar**, **gzip**
```

Comment:
After #808 is merged, the supported **file types** (should change from "supported format") will be zip, tar, gz, gzip, 7z, tgz, z